truffle/test/LoadContract.js

```diff
@@ -18,17 +19,31 @@ contract('LoadContract', async (accounts) => {
     const SHIPPER = accounts[1];
     const CARRIER = accounts[2];
     const MODERATOR = accounts[3];
-    const INVALID = accounts[9];
+    const ATTACKER = accounts[9];
+
+    async function createNotary() {
+        const notaryContract = await VaultNotary.new();
+        return notaryContract;
+    }
 
     let contract;
-    before(async () =>{
-        contract = await LoadContract.deployed();
-    });
+    let notary;
 
     async function createShipment(){
         const shipmentUuid = uuidToHex(uuidv4(), true);
         await contract.createNewShipment(shipmentUuid, EscrowFundingType.NO_FUNDING, 0, {from: SHIPPER});
+        await notary.registerVault(shipmentUuid, "", "", {from: SHIPPER});
         await contract.setCarrier(shipmentUuid, CARRIER, {from: SHIPPER});
+        await notary.grantUpdateHashPermission(shipmentUuid, CARRIER, {from: SHIPPER});
+        await contract.setModerator(shipmentUuid, MODERATOR, {from: SHIPPER});
+        return shipmentUuid;
+    }
+
+    async function createShipment2(){
```

Comment:
Same as my comment above -- I'd like something more clear than "2" for these as I don't understand what the difference is or why we have them (and everyone who reads the source will have the same confusion)